<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
    <item row="1" column="0">
     <widget class="QLabel" name="label1">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>检测时间间隔</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label2">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>触发晃动位移</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spin_dist">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="suffix">
       <string> px</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spin_anitime">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label4">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>指针最大大小</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spin_gap">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label3">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>动画持续时间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spin_maxsize">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="suffix">
       <string> px</string>
      </property>
      <property name="minimum">
       <number>32</number>
      </property>
      <property name="maximum">
       <number>640</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
